packages/typespec-test/test/openai/generated/typespec-ts/package.json

```diff
@@ -103,8 +87,30 @@
   },
   "//sampleConfiguration": {
     "productName": "Azure OpenAI API",
-    "productSlugs": ["azure"],
+    "productSlugs": [
+      "azure"
+    ],
     "disableDocsMs": true,
     "apiRefLink": "https://docs.microsoft.com/javascript/api/@msinternal/openai?view=azure-node-preview"
-  }
+  },
+  "tshy": {
```

Comment:
Is there a reason why we're not doing browser builds as part of RLC generation?

Reply:
I think it's because RLC types is meaningless as they are just generated for better user experience ?